Apps/WebClient/src/Server/Controllers/RobotsController.cs

```diff
@@ -40,7 +35,8 @@ public class RobotsController : Controller
         public RobotsController(IConfiguration configuration)
         {
             this.configuration = configuration;
-            this.robotsContent = this.configuration.GetValue("robots.txt", DefaultRobotsContent);
```

Comment:
We should have a default robots which has the old contents (no indexing) and the asset should be read based on the environment key so that we can deliver different robots as needed.

Reply:
so the default is now whatever is in the file, it still allows the configuration key to be added with whatever custom we want and it should replace the default.

Reply:
so we need to have two checked in, the prod setting and the other environments.